README.md

```diff
@@ -63,12 +63,14 @@ You should also specify settings that will be shared across all the plugin rules
     "formComponents": [
       // Components used as alternatives to <form> for forms, eg. <Form endpoint={ url } />
       "CustomForm",
-      {"name": "Form", "formAttribute": "endpoint"}
+      {"name": "Form", "formAttributes": ["registerEnpoint", "loginEnpoint"]}, // allows specifying multiple properties if necessary
+      {"name": "SimpleForm", "formAttribute": "endpoint"}, // deprecated backwards-compatible version
```

Comment:
i'm not sure we need a new option name here - can we just overload `formAttribute` to be either a string, or an array of strings?

Reply:
We can actually. Not a bad idea.